CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-ledger-node ChangeLog
 
+## 15.2.0 - TBD
+
+### Changed
+- Upgraded `jsonld` to `^5.2.0`.
```

Comment:
@davidlehn @dlongley are we sure this is a major change? @davidlehn seems to be suggesting the canonize outputs are similar between versions and the differences I saw is mostly do to different values in the created timestamp.

Reply:
@dlongley @gannan08 @mattcollier: It appears this could be just a minor update based on the tests (including those added in https://github.com/digitalbazaar/bedrock-ledger-node/pull/105 / https://github.com/digitalbazaar/bedrock-ledger-node/pull/106). Are there other considerations that should make this a major release?  For instance, just to avoid potential unknown issues until a manual version update occurs.

Reply:
@aljones15 In a call it was decided to use a major version. Basically just adds a bit of safety since we don't know how to test for potential unknown issues.